### ISSUE TITLE
Change [#include "DHT.h"] to [#include <DHT.h>], my thought 

### ISSUE THREAD

**Issue by jkadofo** [adafruit/DHT-sensor-library#146]
Hello friends,

I am trying out this "DHT Sensor Library", and started off by using the "DHTtester" example.

The issue is related to this line of code:
https://github.com/adafruit/DHT-sensor-library/blob/428e115b57d172cc484d3bcf3f5d0519c2fb6ef1/examples/DHTtester/DHTtester.ino#L8

**TL;DR:** I think the `#include` directive here must be used with angle brackets (<>) rather than double quotes ("") to reduce the chance of surprises. If this is deemed unnecessary, kindly explain to me. Thanks.

**Details:**
Upon a quick inspection of the "DHTtester" example code, I noticed the library was included as:
`#include "DHT.h"`

That is, the `DHT.h` was included as a source file from current sketch directory rather using angle brackets (<>) to include it as a standard header file as in:
`#include <DHT.h>`

Both versions compile fine for me (even when I have my sketch folder in another location). However I think the angle bracket version would be a better option as that would prevent any surprises, assuming the user somehow had another "DHT.h" file in their working sketch folder.

Furthermore, the other example which is "DHT_Unified_Sensor" uses angle brackets to include the `DHT.h` and other dependencies, as shown below:
https://github.com/adafruit/DHT-sensor-library/blob/428e115b57d172cc484d3bcf3f5d0519c2fb6ef1/examples/DHT_Unified_Sensor/DHT_Unified_Sensor.ino#L11

In conclusion, I think we should use angle brackets for "DHTtester" example. If this is however deemed unnecessary, I would be glad to get an explanation for that so I can learn from it.

Thanks for your hardwork and efforts to put together this library and many others.
Cheers.

**Comment by siddacious** [comment]
If you submit a PR I'll be happy to review it
